pkg/api/rest/types.go

```diff
@@ -60,7 +81,7 @@ type podStrategy struct {
 
 // Pods is the default logic that applies when creating and updating Pod
 // objects.
-var Pods RESTCreateStrategy = podStrategy{api.Scheme, api.SimpleNameGenerator}
+var Pods = podStrategy{api.Scheme, api.SimpleNameGenerator}
```

Comment:
In my later pull I moved this class to `pkg/registry/pod/types.go` as "pod.Strategy".  So this is the default implementation of the basic pod strategy.  I can add the comments here to make that clear.

RESTCreate and RESTUpdate are separate.  There would be a different RESTUpdateStrategy for updating the pods as a regular user (PUT /pods/foo) vs updating status (PUT /pods/foo/status -> RESTStatusUpdateStrategy).  The separate strategy implementation there is just so we can guarantee type safety (the reflective implementation doesn't buy us a lot).

https://github.com/smarterclayton/kubernetes/commit/dc4c9dae656ac90c373ee1dcb5e8d2925dee2deb#diff-189461deaff2db4ed8975a0bd74f9b4fR43